### PR TITLE
Add menus

### PR DESCRIPTION
**Pull request by Jomkit**
Pushing my progress as requested

### CODE REVIEW

File: Assets/Scripts/UI/ButtonTest.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class ButtonTest : MonoBehaviour
```
Comment (toryjleo) [inline Assets/Scripts/UI/ButtonTest.cs:6]: Could you change the name of this class to something that makes a bit more sense? Is this the main menu?

File: Assets/Scripts/UI/PauseMenu.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        pauseMenuUI.SetActive(false);
+        Time.timeScale = 1f;
+        GameIsPaused = false;
+    }
+
+    void Pause()
+    {
+        
+        pauseMenuUI.SetActive(true);
+        Time.timeScale = 0f;
```
Comment (toryjleo) [inline Assets/Scripts/UI/PauseMenu.cs:37]: Could you please remove the timescale changes? Pausing the game time causes bugs.

File: Assets/Scripts/UI/PauseMenu.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        GameIsPaused = true;
+    }
+    public void LoadMenu()
+    {
+        Debug.Log("Loading Menu...");
+    }
+    public void QuitGame()
+    {
+        Debug.Log("Quitting Game...");
```
Comment (toryjleo) [inline Assets/Scripts/UI/PauseMenu.cs:46]: Please add Quitting functionality

File: Assets/Scripts/UI/ButtonTest.cs [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+        TestButton1.onClick.AddListener(fuck);
+        Debug.Log("Fuck");
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        float time = Time.deltaTime;
+    }
+    private void fuck()
```
Comment (toryjleo) [inline Assets/Scripts/UI/ButtonTest.cs:22]: Remove fuck function

File: Assets/Scripts/UI/ChangeScene.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ChangeScene : MonoBehaviour
```
Comment (toryjleo) [inline Assets/Scripts/UI/ChangeScene.cs:6]: What is changescene? Could you add a class description?

example: https://github.com/toryjleo/BikeIdea/blob/master/Assets/Scripts/Player/BikeScript.cs#L8-L9

File: Assets/Scripts/UI/ButtonTest.cs (hunk 1 above)
Comment (Jomkit) [reply Assets/Scripts/UI/ButtonTest.cs:6]: This was just a script for testing button press, I'm gonna remove it

File: Assets/Scripts/UI/PauseMenu.cs (hunk 2 above)
Comment (Jomkit) [reply Assets/Scripts/UI/PauseMenu.cs:37]: I can remove it, but I'm not sure what to replace it with to enable the game to pause when bringing the pause menu up

**Review by LiamMillerSE — APPROVED** [review]

**Review by toryjleo — CHANGES_REQUESTED** [review]
- [ ] Update Main Menu button alignment
 - [ ] Remove changes to TrueScene
 - [ ] Merge with Liam's branch
 - [ ] Add back PauseMenu blueprint

**Comment by LiamMillerSE** [comment]
Play and Quit buttons are high and locked to the sides, when I maximize the game, the buttons are off the top of the screen

**Review by LiamMillerSE — APPROVED** [review]

**Review by toryjleo — APPROVED** [review]

**Pull request merged**